react/features/screen-share/middleware.js

```diff
@@ -3,9 +3,12 @@
 import { CONFERENCE_JOINED } from '../base/conference';
 import { MiddlewareRegistry } from '../base/redux';
 
-import { SET_SCREENSHARE_CAPTURE_FRAME_RATE } from './actionTypes';
+import { SET_SCREENSHARE_CAPTURE_FRAME_RATE, SET_SCREEN_AUDIO_SHARE_STATE } from './actionTypes';
+import AUDIO from './constants';
```

Comment:
The constants file is empty. Also, can't we use `MEDIA_TYPE.AUDIO`?

Reply:
done